pkg/versiongetter/github_release.go

```diff
@@ -95,10 +96,13 @@ func (g *GitHubReleaseVersionGetter) List(ctx context.Context, pkg *registry.Pac
 				})
 			}
 		}
-		if len(releases) != opt.PerPage {
+		if limit > 0 && len(items) >= limit { // Reach the limit
+			return items[:limit], nil
+		}
+		if resp.LastPage == 0 {
```

Comment:
```suggestion
		if resp.NextPage == 0 {
```

Is this correct?
The example refers to `NextPage`.

https://github.com/google/go-github#pagination